src/pages/Register/RegisterForm.jsx

```diff
@@ -0,0 +1,248 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import FormButton from '../../components/FormButton';
+import { listOfStates } from '../../utilities/registerHelpers';
+
+const inputs = JSON.parse(localStorage.getItem('registerForm')) || {
+  firstName: '',
+  lastName: '',
+  email: '',
+  password: '',
+  confirmPassword: '',
+  address1: '',
+  address2: '',
+  city: '',
+  state: '',
+  zip: '',
+  age: '',
+  gender: '',
+};
+
+const requiredFields = [
+  'firstName',
+  'LastName',
+  'email',
+  'password',
+  'address1',
+  'city',
+  'state',
+  'zip',
+  'age',
+  'gender',
+];
+
+const RegisterForm = () => {
+  const navigate = useNavigate();
+
+  const [formInputs, setFormInputs] = useState(inputs);
+
+  useEffect(() => {
+    const token = localStorage.getItem('token');
+    if (token) {
```

Comment:
Not an issue for now, but might consider proving that the token is valid here rather than checking for any ol' token